internal/core/application/feeder/error.go

```diff
@@ -0,0 +1,9 @@
+package feeder
```

Comment:
As you can notice, none of the other app services have any hardcoded error and this is actually intended.
My idea was to dedicate a PR (with prior discussion) to have a common place for all the currently repeated internal types, and also constants and errors. Not high priority at this stage tho.
Therefore, I ask you to drop this file and return `fmt.Errorf` directly in the app service methods.

Reply:
Done